library/X509/Job.php

```diff
@@ -191,16 +194,16 @@ protected function getScanTargets(): Generator
         if (! $this->isRescan()) {
             yield from $this->generateTargets();
         } else {
-            $targets = (new Select())
-                ->columns(['id', 'ip', 'hostname', 'port'])
-                ->from('x509_target');
-
+            $targets = X509Target::on($this->db)->columns(['id', 'ip', 'hostname', 'port']);
             if ($this->sinceLastScan) {
-                $targets->where(new Expression('last_scan < %d', [$this->sinceLastScan->getTimestamp()]));
+                // The filter processor doesn't allow us to filter using expressions
```

Comment:
Is this still true?

Reply:
still? After all, not much has changed since then. But yes, it is still unsupported!

Reply:
And `Filter::greateThan()` does not work? Also `where(['last_scan < ?', $timestamp])` should work. I don't see the need for an expression. Anyway do we have an issue for expression support in our ORM?